features/support/env.rb

```diff
@@ -6,19 +6,25 @@
 
 ENV["RAILS_ENV"] ||= "cucumber"
 
+# The version of rails that gets loaded depends on which Gemfile in gemfiles/ is loaded.
+require 'rails'
+require 'active_record'
+require 'active_admin'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.